helm-chart/templates/04-hub-deployment.yaml

```diff
@@ -28,6 +28,7 @@ spec:
           command:
             - ./hub
             {{ .Values.tap.debug | ternary "- -debug" "" }}
+            {{ .Values.tap.notelemetry | ternary "- -no-telemetry" "" }}
```

Comment:
Instead, use the `TELEMETRY_DISABLED` environment variable mentioned in https://github.com/kubeshark/hub/pull/4#discussion_r1305936600

Reply:
changed to:
```yaml
         command:
            - ./hub
            {{ .Values.tap.debug | ternary "- -debug" "" }}
            - e
            - TELEMETRY_DISABLED={{ .Values.tap.notelemetry }}
```

Reply:
Since outdated, continuing at https://github.com/kubeshark/kubeshark/pull/1420#discussion_r1309452513